SortApp/SortApp.BL.Test/Sorting/BubbleSortTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SortApp.BL.Test.Sorting
+{
+	/// <summary>
+	/// Represents the tests executed towards the <see cref="SortApp.BL.Sortings.BubbleSort{T}"/> sorting class.
+	/// </summary>
+	/// <owner>Oleh Petrenko</owner>
+	[TestClass]
+	public sealed class BubbleSortTests
+	{
+		/// <summary>
+		/// Performs a test towards the logics of bubble sort algorithm when incoming array is filled.
+		/// </summary>
+		/// <owner>Oleh Petrenko</owner>
+		[TestMethod]
+		public void BubbleSortFilledArray()
```

Comment:
Please, Add suffix Test for all you tests names. In this case it's BubbleSortFilledArrayTest